installer/bundle/bundle_skel.sh

```diff
@@ -841,6 +842,19 @@ case "$installMode" in
         fi
 
         if [ $KIT_STATUS -eq 0 ]; then
+            # These steps are required to fix the mysql2 gem compatibility issue on latest Ubuntu distros and can be removed after GA release >2016_v1.2.0-75
+            if [ ${OMS_RUBY_DIR}/gem list | grep -q "mysql2" ]; then
+                # Cleans up static mysql2 ruby gem if it exists in omsagent ruby path
+                ${OMS_RUBY_DIR}/gem uninstall mysql2 1> /dev/null 2> /dev/null
+            fi
+			echo "----- Installing MySQL2 ruby gem -----"
```

Comment:
This indentation (and the lines below) do not appear to be lining up properly, at least in this review.

Please use spaces for indentation so it'll look correct here, thanks.